invenio_jsonschemas/views.py

```diff
@@ -45,10 +48,26 @@ def create_blueprint(state):
     @blueprint.route("/<path:schema_path>")
     def get_schema(schema_path):
         """Retrieve a schema."""
-        try:
-            return send_from_directory(state.get_schema_dir(schema_path),
-                                       schema_path)
-        except JSONSchemaNotFound:
-            abort(404)
+        replace_refs = request.args.get('refs', '')
+        if replace_refs == 'resolve':
+            schema = {'$ref': request.base_url}
+            loader_cls = import_string(
+                current_app.config['JSONSCHEMAS_LOADER_CLS']
+            )
+            try:
+                return jsonify(JsonRef.replace_refs(
```

Comment:
Can we memoize/cache this? We could even produce an etag so that it's cached by clients. Anyway the result to this call won't change on production unless the corresponding package having the schema isn't updated, and in that case we can assume the WSGI processes are reloaded.